client/src/routes/automation/BlocklyComponent.svelte

```diff
@@ -61,41 +61,29 @@
     });
 </script>
 
-<div>
+<div class="pr-4">
     {#await entityPromise}
         <p>{tr('blockly.loading')}</p>
-        <div id="blocklyDivHide" />
+        <div
+            id="blocklyDivHideAwait"
+            class="absolute bg-esidom z-100 w-full h-vh-80"
+        />
     {:then}
-        <p class="py-4">
+        <p class="pb-6">
             <BorderedButton
                 on:click={() => blocklyService.convertToBlock()}
                 text={tr('blockly.convertBlock')}
             />
         </p>
     {:catch}
         <p style="color: red">{tr('blockly.loadingError')}</p>
-        <div id="blocklyDivHide" />
+        <div
+            id="blocklyDivHideError"
```

Comment:
Same here ?

Reply:
not used, just to make the code more readable